packages/support/resources/views/components/actions/group.blade.php

```diff
@@ -7,9 +7,17 @@
     'tooltip' => null,
 ])
 
-<div x-data="{ isOpen: false }" {{ $attributes->class(['relative']) }}>
+<div
+    x-data="actionGroupTableComponent({
+        mountTableActionUsing: async (...args) => {
```

Comment:
Using the same approach as the other components. We need this here because of `$wire`